### ISSUE TITLE
styled-components theme support

### ISSUE THREAD

**Issue by nbar1** [KamiKillertO/vscode-colorize#250]
Would be awesome to have the extension be given a styled-components theme file in the config that it could then match the color to the theme variable.

For instance, this would be colored:

`color: ${props => props.theme.primaryColor};`

**Comment by mateushnsoares** [comment]
I agree with and have another idea above this idea when having a conditional in theme passed to theme provider and you use a color of the theme the colorize add two ( or more ) colors in the same expression like this:

          |          red              |            blue           |
color: ${props => props.theme.primaryColor};
